### DISCUSSION TITLE
Merry Christmas

### DISCUSSION THREAD

**Discussion by kebekus** [Akaflieg-Freiburg/enroute#315] [Team Posts]
Dear all,

the end of the year is near, so let me thank all of you for your contributions to **Enroute Flight Navigation**. With about 11,000 active installations, the app has by now become a visible player in the field. That would not have been possible without your dedication.

Thank you! · Merry Christmas · Happy New Year.

Stefan.

**Comment by TTC70** [comment]
Hello Stefan,
please accept my best greetings for the holidays.
Thank you and thank Ivan for the opportunity to be part of your team.
Merry Christmas and a Happy New Year to all, hoping that 2022 will be
filled with less "pain" and "burden" than 2021.
Ciao ciao
Antonio

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Mail
priva di virus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Il giorno ven 24 dic 2021 alle ore 08:00 Stefan Kebekus <
***@***.***> ha scritto:
